dao/MigrateDAO.php

```diff
@@ -112,13 +112,24 @@ function removeSite($link_id, $user_id, $site_id) {
 
     function startMigration($link_id, $user_id, $site_id, $notifications, $dept, $term, $provider) {
         $now = date("Y-m-d H:i:s");
-        $workflow = ["$now,000 - [INFO] - Started Migration ($site_id)","$now,001 - [INFO] - Scheduled Export..."];
 
-        $query = "UPDATE {$this->p}migration_site
-                SET modified_at = NOW(), modified_by = :userId, started_at = NOW(), started_by = :userId, 
-                    workflow = :workflow, active = 1, state='starting', notification = :notifications,
-                    term =  :term, provider = :provider, dept = :dept, report = NULL, files = NULL, imported_site_id = 0, transfer_site_id = NULL
-                WHERE `link_id` = :linkId and `site_id` = :siteId;";
+        $user_details = $this->PDOX->rowDie("SELECT user_id, displayname,email FROM {$this->p}lti_user WHERE user_id = :userid;",
+                                            array(':userid' => $user_id));
+
+        $user_name = $user_details['displayname'];
+        $user_email = $user_details['email'];
+
+        $workflow = ["$now,000 - [INFO] - Migration for site $site_id started by $user_name ($user_email)","$now,001 - [INFO] - Scheduled Export..."];
```

Comment:
@TurRil  Do we need this "001 ... Schedule Export ..." log line? Doesn't seem to be adding any useful info.